### PR TITLE
Support 1.x setKey & 2.x setKeys 

### LINKED ISSUE #1302: substrate 2.x does not have session.setKey
We should have feature detection for these, for 1.x & 2.x - `session.setKey` for 1.x, `session.setKeys` for 2.x

There are potentially other changes here as well, need to look at them. i.e. there has been some changes to staking.info queries, including retrieval of session keys. 

Some functionality is currently also limited, i.e. we cannot detect session lengths, bonding duration, or anything done via params - until these are exposed

### PR DESCRIPTION
**Pull request by Tbaut**
- closes #1302 
- I did not test on substrate 2.0 yet

### CODE REVIEW

**Comment by jacogr** [comment]
I can test on 2.x

File: packages/app-staking/src/Actions/Account/SetSessionAccount.tsx [hunk 1]
```diff
@@ -66,7 +68,7 @@ class SetSessionKey extends React.PureComponent<Props, State> {
               label={t('Set Session Key')}
               onClick={ onClose }
               params={[sessionId]}
-              tx='session.setKey'
+              tx={api.tx.session.setKey ? 'session.setKey' : 'session.setKeys'}
```
Comment (jacogr) [inline packages/app-staking/src/Actions/Account/SetSessionAccount.tsx]: Would just put the latest, i.e. 2.x first

**Comment by jacogr** [comment]
Needs #1327

**Comment by jacogr** [comment]
Ok, works on 2.x.

**Comment by Tbaut** [comment]
I noticed a similar issue with the signer not showing the actual values passed as argument. Will investigate.

**Comment by Tbaut** [comment]
Should be good to go, the problem mentioned above is unrelated and already present in master. Logged https://github.com/polkadot-js/api/issues/1060

**Pull request merged**